releasenotes/notes/unify-instructions-and-commands-aaa6d8724b8a29d3.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I'm going to update this particular note with the related changes to give a complete picture of this migration.